openghg/plotting/_timeseries.py

```diff
@@ -1,61 +1,191 @@
-from typing import List, Optional, Union
-
+from typing import List, Optional, Union, Dict, Tuple
 import plotly.graph_objects as go
+import numpy as np
+import base64
+
 from openghg.dataobjects import ObsData
+from openghg.util import get_species_info, synonyms, get_datapath
+
+
+def _latex2html(
+    latex_string: str
+) -> str:
+    """Replace latex sub/superscript formatting with html.
+    Written because the latex formatting in Plotly seems inconsistent
+    (works in Notebooks, but not VC Code at the moment).
+
+    Args:
+        latex_string: 
```

Comment:
Documentation string is not quite complete here.

Reply:
Yeah this needs completing.

Reply:
Added details to doc-string